libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/Notification.java

```diff
@@ -93,6 +86,18 @@ public NotificationEmail getNotificationAsEmail() {
         return null;
     }
 
+    public Notification setNotificationToMetricConverter(NotificationToMetricConverter notificationToMetricConverter) {
+        this.notificationToMetricConverter = notificationToMetricConverter;
+        return this;
+    }
+
+    public NotificationMetric getNotificationAsMetrics(Timestamp currentTime) {
```

Comment:
I get the currentTime as an interface argument so equality checks will be successful in tests (calculating the currentTime inside each notification makes a difference for attributes such as "expiry_days" which can sometimes result in test failures)